pkg/util/retryutil/retry_util.go

```diff
@@ -17,8 +17,10 @@ type Ticker struct {
 	ticker *time.Ticker
 }
 
+// Stop returns ticker.Stop
 func (t *Ticker) Stop() { t.ticker.Stop() }
 
+// Tick returns ticker.C
```

Comment:
Let's give it more meaningful commentary, e.g. Wrapper around ticker to make it testable.